app/src/main/res/values/font_certs.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <array name="com_google_android_gms_fonts_certs">
+        <item>@array/com_google_android_gms_fonts_certs_dev</item>
```

Comment:
we can find these credentials from https://github.com/android/compose-samples/blob/main/Jetchat/app/src/main/res/values-v23/font_certs.xml as mentioned in the docs here https://developer.android.com/develop/ui/compose/text/fonts#downloadable-fonts